delta/plugins/elasticsearch/src/test/resources/query/query-schema-deprecation-types.json

```diff
@@ -2,14 +2,26 @@
   "query": {
     "bool": {
       "must_not": {
-        "term": {
-          "@type": "https://bluebrain.github.io/nexus/vocabulary/CrossProject"
+        "bool": {
```

Comment:
I think it better to delete the related test, the important is to test the output, not to test the query